arbor/include/arbor/cable_cell.hpp

```diff
@@ -16,22 +17,17 @@
 
 namespace arb {
 
-// Current clamp description for stimulus specification.
-
-struct i_clamp {
-    using value_type = double;
-
-    value_type delay = 0;      // [ms]
-    value_type duration = 0;   // [ms]
-    value_type amplitude = 0;  // [nA]
-
-    i_clamp(value_type delay, value_type duration, value_type amplitude):
-        delay(delay), duration(duration), amplitude(amplitude)
-    {}
+// Pair of indexes that describe range of local indices.
+// Returned by cable_cell::place() calls, so that the caller can
+// refer to targets, detectors, etc on the cell.
+struct locrange {
+    cell_lid_type first;
+    cell_lid_type last;
+    locrange(cell_lid_type first, cell_lid_type last):
+        first(first), last(last) {}
 };
```

Comment:
I don't like using `std::pair` for public API code. `second` is even more misleading than `last`.

I have changed it to `lid_range`, and using `begin` and `end` instead of `first` and `last`.